hack/util.sh

```diff
@@ -94,20 +97,26 @@ function start_os_server {
       --master-config=${MASTER_CONFIG_DIR}/master-config.yaml \
       --node-config=${NODE_CONFIG_DIR}/node-config.yaml \
       --loglevel=4 \
-      1>&2 2>"${BASETMPDIR}/openshift.log" &
+      1>&2 2>"${LOG_DIR}/openshift.log" &
```

Comment:
I did not remove ... I just switched the location of the openshift.log file from the base temp dir to the log dir.